### PR TITLE
Requires grad context manager

### PR DESCRIPTION
**Pull request by samedii**

### CODE REVIEW

File: workflow/torch/requires_grad.py [hunk 1]
```diff
@@ -2,19 +2,28 @@
 
 
 @contextmanager
-def requires_nograd(module):
+def requires_grad(module, grad=True):
```
Comment (Jim-Holmstroem) [inline workflow/torch/requires_grad.py]: I would assert that grad is bool, could be kinda tricky bug to hunt down

**Comment by Jim-Holmstroem** [comment]
really nice PR :)

**Review by FelixAbrahamsson — APPROVED** [review]

File: workflow/torch/requires_grad.py (hunk 1 above)
Comment (samedii) [reply workflow/torch/requires_grad.py]: I'll add an assert when the other branch is merged into this

Comment (samedii) [reply workflow/torch/requires_grad.py]: Added check on grad

**Review by Jim-Holmstroem — APPROVED** [review]

**Pull request merged**